index.html

```diff
@@ -36,16 +36,16 @@ <h1>Hey there. I’m Salim.<br><span class="highlight">I’m a software develope
                 </p>
             </div>
             <div class="social-media">
-                <a href="#"><img src="./static/images/Icons/github.png" alt="github-image"></a>
-                <a href="#"><img src="./static/images/Icons/linkedIn.png" alt="linkedIn-image"></a>
-                <a href="#"><img src="./static/images/Icons/angel_list.png" alt="angel_list-image"></a>
-                <a href="#"><img src="./static/images/Icons/twitter.png" alt="twitter-image"></a>
-                <a href="#"><img src="./static/images/Icons/medium.png" alt="medium-image"></a>
+                <li><a href="#"><img src="./static/images/Icons/github.png" alt="github-image"></a></li>
+                <li><a href="#"><img src="./static/images/Icons/linkedIn.png" alt="linkedIn-image"></a></li>
+                <li><a href="#"><img src="./static/images/Icons/angel_list.png" alt="angel_list-image"></a></li>
+                <li><a href="#"><img src="./static/images/Icons/twitter.png" alt="twitter-image"></a></li>
+                <li><a href="#"><img src="./static/images/Icons/medium.png" alt="medium-image"></a></li>
             </div>
```

Comment:
Good job :clap:  adding the `li` tag per icon but As said by the previous reviewer ,The list of icons should use a `ul` tag to group them,this means that you need to replace that `div` with `social-media` class with a `ul` tag, Because the `li` is not allowed as child of element `div` tag, That may even fix your linter errors :smiley: 

![image](https://user-images.githubusercontent.com/75126481/138176551-9c283e30-a2c2-49ee-91b3-cbeefc007ef5.png)

Reply:
You are welcome :smiley:  Happy coding :computer: